app/onboarding.md

```diff
@@ -0,0 +1,44 @@
+# Technician onboarding
```

Comment:
I was wondering if you could write your onboarding guide as a blog post and publish it, so that we can link back to it from the community materials?

dev.to gives free publishing and is useful if you don't have a personal site yet